CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Upcoming
```

Comment:
The commit name is not in the imperative form 😱

Reply:
Fixed!